courses/courseTable.h

```diff
@@ -16,7 +16,7 @@ struct CourseTable {
     CourseVtx *vertexStart;    // 0x18 segmented address of vertex data
     u32 vertexCount;    // 0x1C number of vertices in vertex data
     u8 *packedStart;    // 0x20 packed display list start address
-    uintptr_t finalDisplaylistOffset;        // 0x24
+    u8* finalDisplaylistOffset;        // 0x24
```

Comment:
This is *technically* a uintptr_t in that it contains an addr. Is it possible to do uintptr_t * or uintptr_t?

Reply:
Perhaps it should be `Gfx *` since it's technically a ptr to some gfx. I just think that's sort of confusing in a sense cause it's not really a ptr to a displaylist. But tomato tomato?

Reply:
uintptr_t * or Gfx * I'm good with if either of those works.

Reply:
I think Gfx* is better